psi4/src/read_options.cc

```diff
@@ -226,6 +226,8 @@ int read_options(const std::string &name, Options & options, bool suppress_print
   /*- List of basis function indices for which cube files are generated
   (1-based). All basis functions computed if empty.-*/
   options.add("CUBEPROP_BASIS_FUNCTIONS", new ArrayType());
+  /* Fraction of density captured by adaptive isocontour values */
+  options.add_double("CUBEPROP_ISOCONTOUR_THRESHOLD",0.85);
```

Comment:
I feel like we should be standardizing these names. There's precedent for `_PERCENTAGE` (though no consensus on 0-->1, 0-->100). I actually like `_FRACTION` better, but there's no precedent for that. There are no `_THRESHOLD`. I'm fine with it as-is, but if either of the former appeal, go for it (sticking with 0--1, imo).

Reply:
I like `_FRACTION`, will fix when free this week.